src/schema/nmdc.yaml

```diff
@@ -255,16 +255,23 @@ classes:
 
   FunctionalAnnotationAggMember:
     class_uri: nmdc:FunctionalAnnotationAggMember
+    unique_keys: 
```

Comment:
leaving a note here to make sure @turbomam or @sierra-moxon review this use of unique_keys (in response to #2155)

Reply:
Have we discussed this at all? @kheal do you still want to discuss the use of `unique_keys` even though the PR is closed?

Reply:
I have moved this PR to the nmdc-schema repo: https://github.com/microbiomedata/nmdc-schema/pull/2203.  I'd appreciate input on the `unique_keys` use there.